src/main/resources/templates/create-user.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html PUBLIC "http://www.thymeleaf.org/dtd/xhtml1-strict-thymelefa-4.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml"
+      xmlns:th="http://www.thymeleaf.org">
+<head>
+    <title>Finance Friend Login</title>
+    <link rel="stylesheet" type="text/css" />
+</head>
+
+<body>
+
+<div>
+    <img th:src="@{/images}" th:alt-title="Finance Friend Logo">
```

Comment:
This isn't actually pointing at an image -- need something like /images/myImage.png